lib/arjdbc/postgresql/base/array_encoder.rb

```diff
@@ -0,0 +1,21 @@
+# This implements a basic encoder to work around ActiveRecord's dependence on the pg gem
+module PG
```

Comment:
I'm not sure if this is the best way to do the array encoding or not. I couldn't figure out how it was previously being done, so if you have info on that and would like me to try to go that route I can dig into it. The one difference with this way that I noticed is that all of the values are always quoted inside the array, even if they are numbers. It doesn't seem like that matters based on the tests but it is something to keep in mind.